src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,682 @@
+#### provider
+
+<p>
+
+**Type:** `AstroFontProvider` | `"local"`
+</p>
+
+The source of your font files. You can use a [built-in provider](#available-remote-font-providers), write your own [custom provider](#custom-font-providers), or set to `"local"` to use local font files:
+
+```js title="astro.config.mjs"
+import { defineConfig, fontProviders } from "astro/config"
+
+export default defineConfig({
+    experimental: {
+        fonts: [{
+            provider: fontProviders.google(),
+        }]
+    }
+})
+```
+
+#### name
+
+<p>
+
+**Type:** `string`
+</p>
+
+The font family name, as identified by your font provider:
+
+```js title="astro.config.mjs"
+name: "Roboto"
+```
+
+#### cssVariable
+
+<p>
+
+**Type:** `string`
+</p>
+
+An identifier of your choice for your font, conventionally in the form of a CSS variable. It must be a [valid `<ident>`](https://developer.mozilla.org/en-US/docs/Web/CSS/ident):
+
+```js title="astro.config.mjs"
+cssVariable: "--font-roboto"
+```
+
+#### fallbacks
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["sans-serif"]`
+</p>
+
+An array of fonts to use when your chosen font is unavailable, or loading:
+
+```js title="astro.config.mjs"
+fallbacks: ["CustomFont", "serif"]
+```
+
+If the last fallback is a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name), Astro will try to generate an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics.
+
+#### automaticFallback
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `true`
+</p>
+
+Whether or not to enable automatic fallback font generation:
+
+```js title="astro.config.mjs"
+automaticFallback: false
+```
+
+### Remote font properties
+
+Further configuration options are available for remote fonts. Set these to customize the data loaded from your font provider, for example to only download certain font weights or styles.
+
+Under the hood, thse options are handled by [unifont](https://github.com/unjs/unifont/).
+
+#### weights
+
+<p>
+
+**Type:** `(number | string)[]`<br />
+**Default:** `[400]`
+</p>
+
+An array of [font weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight):
+
+```js title="astro.config.mjs"
+weights: [200, "300", "bold"]
+```
+
+If the associated font is a [variable font](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_fonts/Variable_fonts_guide), you can specify a range of weights:
+
+```js title="astro.config.mjs"
+weights: ["100 900"]
+```
+
+#### styles
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["normal", "italic"]`
+</p>
+
+An array of [font styles](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style):
+
+```js title="astro.config.mjs"
+styles: ["normal", "oblique"]
+```
+
+#### subsets
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["cyrillic-ext", "cyrillic", "greek-ext", "greek", "vietnamese", "latin-ext", "latin"]`
+</p>
+
+An array of [font subsets](https://fonts.google.com/knowledge/glossary/subsetting):
```

Comment:
Gotta find a better link

Reply:
Ooh, this link even allows you to enter a Google font, and it will return all possible subsets with their size!

https://knaap.dev/posts/font-subsetting/ How about this one?